keyboards/novelkeys/nk87/keymaps/via/rules.mk

```diff
@@ -1 +1,8 @@
+
+# Build Options
+#   chagne yes to no to disable
+#
 VIA_ENABLE = yes
+
+# For current implementation it should be set to "yes"
```

Comment:
```suggestion
VIA_ENABLE = yes
```

Reply:
Suggestion was committed and tested using `via` and `default` keymaps.